2016-06-28-Rust-Reach-Fractal.md

```diff
@@ -0,0 +1,91 @@
+
+![Tessel 2 block diagram](http://softdroid.net/sites/default/files/resize/remote/eb0193ce7bae91d87724d28642c83ab7-500x458.png)
+
+This takes OSHW beyond just licensing. We want you to take our designs and use the parts of them you need in your own products. With your help, we can push the boundaries of what OSHW is: a platform for serious product development, not just a sharing marketplace for makers.
+
+> Openness promotes innovation. Tessel is fully open source, hardware and software, so that developers can create and build on these efforts, as projects and as products, without worry of legal hassle. This project seeks to expand and promote openness as a movement in both software and hardware, and be a leader in community developed hardware.
+
+### At the crossroads
+
+Now that we've shipped Tessel 2, there are several things we could do:
+
+1. Keep adding features in software, like support for more languages or additional features
+1. Build more modules to expand the platform
+1. Work on a "Tessel 3" that is an evolution of the board
+1. Work on a "Tessel X" that uses some of the same tooling but suits a different niche
+1. Something else entirely, which approaches the mission in a different way
+
+This is where the Steering Committee comes in. It's our job, as leaders of the project, to determine the vision and create a plan for future development.
+
+For us, this is the first major vision-type decision we've had to make since moving to an all-volunteer open governance model. It's challenging to craft cohesive vision in our limited face-to-face meeting time.
+
+To make a decision of this type, we need to lean back on our mission and core principles, on the logistical implications of our decisions, and on the needs of our community.
+
+Critically, we need to pick projects that are well-scoped and highly focused, so that we can progress measurably with the time and person-power we have available. All of these projects are arguably worthwhile. Which is best?
+
+## Fulfilling the vision
+
+Coming back to the core, we want to build the OSHW movement through our platform. This problem fits the original vision for Tessel 2, to help people of different backgrounds create products quickly and easily.
+
+We've come some of the way towards making Tessel 2 the base for a product you can take to market. But we're not all the way there yet. Here are some of the issues we see:
+
+1. It's not easy for most of our users to take the tools we've created so far and go to market. If you don't come from an electrical engineering background, the next step might be insurmountably daunting. You might not know what things you can afford to remove from the design, or how much efficiency or cost they would save you.
```

Comment:
Could possibly remove the possessive here as well? Or instead say "most of the Tessel community" because they're more than just "users", they contribute to the community by buying and building with Tessel.